cms/djangoapps/contentstore/utils.py

```diff
@@ -442,7 +442,7 @@ def get_taxonomy_list_url():
     return taxonomy_list_url
 
 
-def get_taxonomy_tags_widget_url(course_locator) -> str:
+def get_taxonomy_tags_widget_url(course_locator=None) -> str:
```

Comment:
It looks like this can return `str | None`, not always `str`

Reply:
Updated [here](https://github.com/openedx/edx-platform/pull/33645/commits/7aa790d88e6b4a0fbe0e00c9550d8e5862e767b1#diff-4c3d3ca903f6c372776d85ee0afd1d56fcd920cb2e7d2f207aca45afe9377af5R446).

I used `Optional`, because we need a higher version of Python to use `str | None`

Reply:
That's fine, but you could have just put `from __future__ import annotations` and then `str | None` works.